run_game.py

```diff
@@ -0,0 +1,11 @@
+import sys
+
+from src.main import main
+
+# The major, minor version numbers your require
+MIN_VER = (3, 9)
```

Comment:
can the game run on older versions, 3.7 or 3.6 even? Should use min version it works

Reply:
Last PyWeek, there were issues on 3.8 (Sound if I recall correctly)